.platform/hooks/prebuild/01_software_dependencies.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Install Node 12.16.1
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
+source ~/.nvm/nvm.sh
+nvm install 12.16.1
+
+# Install Frontend dependencies
+npm --prefix src/ts ci
```

Comment:
are we calling this file whenever we're deploying?

Reply:
Yes, I should edit this script a bit so we aren't doing anything redundant on the instances.

Reply:
changed!